tests/phpunit/inventory/BaseInventoryTest.php

```diff
@@ -85,4 +85,18 @@ public function testAddMultipleItemsInOneCallWithLeftover() : void{
 		}
 		self::assertSame(20, $leftoverCount);
 	}
+
+	public function testAddItemWithLargeQuantity() : void{
+		$inventory = new class(10) extends SimpleInventory{
+
+		};
+		$inventory->addItem(VanillaItems::APPLE()->setCount(100));
```

Comment:
You should assert that the number of items returned from `addItem()` is zero as well.